skimage/measure/profile.py

```diff
@@ -109,3 +122,222 @@ def _line_profile_coordinates(src, dst, linewidth=1):
+            # Prevent empty arrays from being added, when a center point is detected for example
+            if len(points_array) > 0:
+                sampling_array.append(points_array)
+
+    # Return transposed array for ndi.map_coordinates
+    return np.array(sampling_array, dtype=float).T
+
+
+def _distance_point_line_3d(point, src, dst):
+    """Return the distance between a line and a point in 3d.
+
+    Parameters
+    ----------
+    point : 3-tuple of numeric scalar (float or int)
+        The point to find the distance.
+    src : 3-tuple of numeric scalar (float or int)
+        A first point where the line is passing through
+    dst : 3-tuple of numeric scalar (float or int)
+        A second point where the line is passing through
+
+    Returns
+    -------
+    point : float
+        The distance between the point and the line in units
+
+    """
+
+    line_vector = np.subtract(dst, src)
+    length_line = np.linalg.norm(line_vector)
+    vector1 = np.subtract(point, dst)
+    vector2 = np.subtract(point, src)
+    distance = np.linalg.norm(np.cross(vector1, vector2) / length_line)
+    return np.abs(distance)
+
+
+def _rotate_point_around_line(point_to_rotate, point_on_line, unit_direction_vector, angle_in_radians):
+    """Return a 3d point that is rotated at an angle of theta around a line
+    passing through a selected point.
+
+    Parameters
+    ----------
+    point_to_rotate : 3-tuple of numeric scalar (float or int)
+        The point to rotate.
+    point_on_line : 3-tuple of numeric scalar (float or int)
+        A point where the line is passing through
+    unit_direction_vector : 3-tuple of numeric scalar (float or int)
+        The unit direction vector of the line
+    unit_direction_vector : float or int
+        The angle of rotation in radians
+
+    Returns
+    -------
+    point : array, shape (3), float
+        The coordinates of the rotated point around the line
+
+    """
+
+    a = point_on_line[0]
+    b = point_on_line[1]
+    c = point_on_line[2]
+
+    x = point_to_rotate[0]
+    y = point_to_rotate[1]
+    z = point_to_rotate[2]
+
+    u = unit_direction_vector[0]
+    v = unit_direction_vector[1]
+    w = unit_direction_vector[2]
+
+    p1 = (a * (v ** 2 + w ** 2) - u * (b * v + c * w - u * x - v * y - w * z)) * (
+        1 - np.cos(angle_in_radians)) + x * np.cos(
+            angle_in_radians) + (-c * v + b * w - w * y + v * z) * np.sin(angle_in_radians)
+
+    p2 = (b * (u ** 2 + w ** 2) - v * (a * u + c * w - u * x - v * y - w * z)) * (
+        1 - np.cos(angle_in_radians)) + y * np.cos(
+            angle_in_radians) + (-c * u - a * w + w * x - u * z) * np.sin(angle_in_radians)
+
+    p3 = (c * (u ** 2 + v ** 2) - w * (a * u + b * v - u * x - v * y - w * z)) * (
+        1 - np.cos(angle_in_radians)) + z * np.cos(
+            angle_in_radians) + (-b * u + a * v - v * x + u * y) * np.sin(angle_in_radians)
+
+    return [p1, p2, p3]
```

Comment:
Docstring specifies an array, so either this or the docstring needs to be changed.